content/blog/2024-07-21-RLadies-Bioconductor-collab/index-en.qmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "Invitation to participate in the new R-Ladies–Bioconductor collaboration!"
+description: |
+  Announcing a new collaboration between R-Ladies and Bioconductor
+author:
+  - name: Janani Ravi
+    url: https://github.com/jananiravi
+date: 2024-07-21
+categories:
+  - RLadies
+  - Bioconductor
+image: "rladies-bioc.png"
+editor: 
+  markdown: 
+    wrap: sentence
+---
+
+![](rladies-bioc.png){ width=50% fig-align="center"}
+
+We are excited to invite you to join a new collaboration between two global communities, each committed to building diverse developer communities around the world, R-Ladies and Bioconductor.
+
+[R-Ladies](http://rladies.org) is a worldwide organization whose mission is to promote gender diversity in the R community.
+Starting in 2012 in San Francisco, USA, R-Ladies now has expanded to more than 200 chapters spanning 60+ countries around the world.
+R-Ladies supports endeavors with other like minded communities such as [LatinR](https://latin-r.com/en/), [RainbowR](https://rainbowr.netlify.app/), [miR](https://mircommunity.com/), [ArabR](https://arabr.github.io), [AfricaR](https://r4africa.org/), and [AsiaR](https://github.com/asiar-community).
```

Comment:
We should mention other communities without them knowing we are going to mention them.

Reply:
We are just mentioning that R-Ladies works with these communities. Do you want me to check in with them?

Reply:
I think we are mixing here the work of the chapters with the work of R-Ladies Global.  

For example, LatinR hosts tutorials with R-Ladies Chapters in Latin America, but R-Ladies Global never officially got involved in that. Now, R-Ladies Global has worked with useR! and Posit, beyond the local chapters. 

I don't know what we have done with the other communities listed there. Even when some of us belong to more than one community, that doesn't mean we do that work representing or in the name of R-Ladies. I would like to give credit for the collaborative work to the chapters when it is due.

Reply:
Agreed that the concept is muddy. Can be resolved by deleting this sentence, as not essential to the scope.